src/api/ums/umsModule.ts

```diff
@@ -128,6 +128,27 @@ export class UMSModule<
     );
   }
 
+  /**
+   * Check if the user is logged in
+   * By default it checks on the token that is marked as DEFAULT otherwise it will use the given alias
+   *
+   * @param alias {string}
+   */
+  public isLoggedIn(alias?: string): Observable<boolean> {
+    const validatedAlias = this.tokenManager.validateTokenAlias(alias);
+
+    if (!validatedAlias) {
+      return of(false);
```

Comment:
Shell it be false or null?

Reply:
yes it should be `false` as the user need to check on a `bool` value. Otherwise the result to the end user will be `true` or `null`...